src/css/index.scss

```diff
@@ -0,0 +1,11 @@
+$moduk-brand-colour: #532a45 !default;
+$moduk-font-family: 'Hevetica Neue', 'Arial', 'sans-serif' !default;
+
+@use 'govuk-frontend/govuk/all' with (
+  $govuk-brand-colour: $moduk-brand-colour,
+  $govuk-font-family: $moduk-font-family
+);
+
+.moduk.govuk-tag {
```

Comment:
Looks good I wonder if its a good idea to split these into different files

Reply:
What your thoughts Im ok to see how it goes and refactor a bit later

Reply:
Yeah I think it's probably a good idea to start doing it now – will have a look and add a fixup. Could be we end up changing the directory structure later when we start working on packaging also.

Reply:
Hmm I'll have a look in the other PR actually, since it has the end-to-end testing etc. in so we can make sure it works :smile: